src/views/home/UserInput.js

```diff
@@ -10,6 +18,39 @@ const urlInputStyle = {
   margin: '10px',
 };
 
+const userAgentOptions = [
+  {
+    key: 'Chrome 70.0.3538.77',
```

Comment:
I feel like different chrome variants wouldn't be as useful as iPhone / iPad / Android phone / Android tablet / Samsung device etc. [This gist](https://gist.github.com/jankeromnes/eb6d371682dc2ed618e6) looks like a pretty good starting point.

Reply:
^^ Going off William I think the dropdown should encapsulate width, height, and user agent, showing up as "devices" (like Apple iPhone X), letting the user pick the type of device and setting the state behind the scenes with that device's corresponding screenshot, width, and user agent. There could then be an additional custom option where the user specifies the width, height, and user agent, which is predominantly what we have now and I'm guessing wouldn't be used nearly as much as simply selecting a device from our list

Reply:
That looks good! Does this mean that the we should take the height and width from the that device.js and not allow the user to enter their own height and width when they have chose a user agent?

Reply:
Yeah, I'm thinking a user would want to choose an entire device instead of only choosing a user agent. And in the case that they want custom functionality they could enter their user agent, width, and height... we could also ask the Target people what they prefer that's just my assumption

Reply:
Yeah I think the select from defaults (devices) / custom makes a lot of sense. It solves the entire ask from Target, and is a much nicer experience for the user - who would probably end up looking up these things anyway.

Reply:
That sounds great! Should users be allowed to choose multiple user agents, just like the custom screen shot size?

Reply:
Hmmm I feel like a user would add multiple devices rather than multiple user-agents for one device, so I think the plus button should add a device, and within each device they are requesting, they can either pick a preconfigured device (user agent, width, height), or specify a custom device (user agent, width and height)... what do u think?

Reply:
Sounds good to me!